modules/nw-ptp-configuring-linuxptp-services-dual-port-oc.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-linuxptp-services-oc-dual-port_{context}"]
+= Configuring linuxptp services as an ordinary clock with dual-port NIC redundancy
+
+You can configure `linuxptp` services (`ptp4l`, `phc2sys`) as an ordinary clock with dual-port NIC redundancy by creating a `PtpConfig` custom resource (CR) object.
+In a dual-port NIC configuration for an ordinary clock, if one port fails, the standby port takes over, maintaining PTP timing synchronization.
+
+:FeatureName: Configuring linuxptp services as an ordinary clock with dual-port NIC redundancy
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator.
+* The node uses an x86 architecture with a dual-port NIC.
```

Comment:
```suggestion
* The node uses an x86_64 architecture with a dual-port NIC.
```

If correct in the given context: https://redhat-documentation.github.io/supplementary-style-guide/#x86_64